### PR TITLE
Add `sentry-graphql`.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

**Comment by maciejwalkowiak** [comment]
Will be finalized once https://github.com/getsentry/sentry-docs/pull/4385 gets merged and we agree on the docs url.

File: packages/maven/io.sentry/sentry-graphql/5.4.0.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "io.sentry:sentry-graphql",
+  "canonical": "maven:io.sentry:sentry-graphql",
+  "version": "5.4.0",
+  "package_url": "https://search.maven.org/artifact/io.sentry/sentry-graphql",
+  "repo_url": "https://github.com/getsentry/sentry-graphql",
+  "main_docs_url": "https://docs.sentry.io/platforms/java/performance/instrumentation/graphql"
```
Comment (marandaneto) [inline packages/maven/io.sentry/sentry-graphql/5.4.0.json:7]:
```suggestion
  "main_docs_url": "https://docs.sentry.io/platforms/java/configuration/integrations/graphql"
```

**Comment by marandaneto** [comment]
@maciejwalkowiak we'd need to generate the 5.4.1 version tho, added the new docs link as a suggestion

**Comment by maciejwalkowiak** [comment]
5.4.1 added. Take a look please if all is fine.

**Comment by marandaneto** [comment]
`repo_url` is wrong but pre-approving it.

**Review by marandaneto — APPROVED** [review]
LGTM

**Review by marandaneto — APPROVED** [review]

**Pull request merged**